### PR TITLE
bpo-33125: Add support for building and releasing Windows ARM64 packages

### PR DESCRIPTION
**Pull request by zooba**
https://bugs.python.org/issue33125

### CODE REVIEW

File: Lib/test/support/__init__.py [hunk 1]
```diff
@@ -1068,7 +1068,7 @@ def change_cwd(path, quiet=False):
     """
     saved_dir = os.getcwd()
     try:
-        os.chdir(path)
+        os.chdir(os.path.realpath(path))
```
Comment (serhiy-storchaka) [inline Lib/test/support/__init__.py]: Why is this needed?

Comment (zooba) [reply Lib/test/support/__init__.py]: @serhiy-storchaka This ensures that the path is fully normalised, in particular, converting shortened paths to long paths. It has an impact during tests later on if the current username is longer than 8 characters.

But I think this change has been merged elsewhere already. When I rebase, it'll probably go away.

**Comment by zooba** [comment]
See bpo-33125 for info about when we'll backport.
